src/view/gui/AppearancePage.cpp

```diff
@@ -170,3 +151,55 @@ void AppearancePage::Restore()
     ApplicationPreferences::GetInstance()->DisplayPreferences();
```

Comment:
Have you tested the functionality to restore ? I guess this change would have broken it.